core-services/image-mirroring/supplemental-ci-images/mapping_supplemental_ci_images_ci

```diff
@@ -66,3 +66,4 @@ quay.io/ocs-dev/odf-console-ci-runner:current registry.ci.openshift.org/ci/odf-c
 quay.io/ocs-dev/odf-console-ci-runner:latest registry.ci.openshift.org/ci/odf-console-ci-runner:latest
 quay.io/strimzi-qe/strimzi-tests:release-0.29.x registry.ci.openshift.org/ci/strimzi-tests:release-0.29.x
 quay.io/strimzi-qe/strimzi-tests:release-0.32.x registry.ci.openshift.org/ci/strimzi-tests:release-0.32.x
+quay.io/redhat_msi/ocp-addons-operators-cli:latest registry.ci.openshift.org/ci/ocp-addons-operators-cli:latest
```

Comment:
Create a separate PR for mirroring this image like [this](https://github.com/openshift/release/pull/37512/files) and get it merged, then only you can use this image in your config file and jobs to run pj-rehearsal. [here](https://docs.ci.openshift.org/docs/how-tos/external-images/#mirror-public-images:~:text=the%20changes%20to%20the%20files%20have%20to%20be%20merged%20before%20the%20images%20can%20be%20used%20in%20jobs%20and/or%20pull%20request%20rehearsals) is the documentation provided.

Reply:
Thanks @ascerra @chetna14manku , this PR is rebased on #37752 for mirroring `ocp-addons-operators-cli` image, will open a seperate PR for `quay.io/modh/ods-ci:latest` as well.
this PR will be updated as soon as those will be merged